components/ee/payment-endpoint/test/fixtures/downgrade_unlimited_to_professional_new/01.downgrade-change-actual.json

```diff
@@ -0,0 +1,206 @@
+{
+  "event_type": "subscription_changed",
+  "content": {
+    "subscription": {
+      "id": "1mkVvmiRMxfTZj1JGs",
+      "customer_id": "643ac637-74ae-4f82-9a86-c8527eb1e496",
+      "plan_id": "professional-new-eur",
+      "plan_quantity": 1,
+      "plan_unit_price": 2300,
+      "plan_amount": 2300,
+      "billing_period": 1,
+      "billing_period_unit": "month",
+      "plan_free_quantity": 0,
+      "status": "active",
+      "current_term_start": 1578406678,
+      "current_term_end": 1581085078,
+      "next_billing_at": 1581085078,
+      "created_at": 1554643078,
+      "started_at": 1554643078,
+      "activated_at": 1554643078,
+      "created_from_ip": "37.173.245.142",
+      "updated_at": 1578406681,
+      "has_scheduled_changes": false,
+      "resource_version": 1578406681833,
+      "deleted": false,
+      "object": "subscription",
+      "currency_code": "EUR",
+      "due_invoices_count": 0,
+      "mrr": 2917,
+      "exchange_rate": 1.0,
+      "base_currency_code": "EUR"
+    },
+    "customer": {
+      "id": "643ac637-74ae-4f82-9a86-c8527eb1e496",
+      "first_name": "Jan",
+      "last_name": "Keromnes",
+      "email": "jan.keromnes@typefox.io",
```

Comment:
IMO the name and email are the only things we need to purge here. Everything else is random test data.